src/App.js

```diff
@@ -27,11 +40,15 @@ export default function App() {
           learn how to code!)
         </p>
         <p>
-          Please type in <SpanCode>pwd</SpanCode>, <SpanCode>ls</SpanCode> or <SpanCode>cd</SpanCode> to get further
+          Please type in <CodeSpan>pwd</CodeSpan>, <CodeSpan>ls</CodeSpan> or <CodeSpan>cd</CodeSpan> to get further
           Information about this specific command in the zshell:{' '}
```

Comment:
I don't think you need the whitespace here? 
```suggestion
          Information about this specific command in the zshell:
```